cpp/ql/src/semmle/code/cpp/Enum.qll

```diff
@@ -57,7 +73,18 @@ class LocalEnum extends Enum {
 }
 
 /**
- * A C++ enum that is declared within a class.
+ * A C++ enum that is declared within a class. For example, the type
```

Comment:
"C++" -> "C/C++".

Reply:
Because it could be enclosed in a struct?

Done.

Reply:
Yes.